pkg/local_object_storage/blobstor/blobovniczatree/blobovnicza.go

```diff
@@ -100,7 +100,12 @@ func NewBlobovniczaTree(opts ...Option) (blz *Blobovniczas) {
 	cache, err := simplelru.NewLRU(blz.openedCacheSize, func(key interface{}, value interface{}) {
 		p := key.(string)
 		lvlPath := filepath.Dir(p)
-		if b, ok := blz.active[lvlPath]; ok && b.ind == u64FromHexString(filepath.Base(p)) {
+
+		blz.activeMtx.RLock()
+		defer blz.activeMtx.RUnlock()
+		b, ok := blz.active[lvlPath]
+
+		if ok && b.ind == u64FromHexString(filepath.Base(p)) {
```

Comment:
The doc specifically says that `activeMtx` must be taken on the caller side when eviction is possible.
With this change we _will_ have a deadlock.

Reply:
Actually, tests most likey failed because of this.

Reply:
oh, i got it, dropped that commit. i hate that code more and more. how do we plan to handle automated eviction then?

Reply:
1. Eviction occurs only with some user operations.
2. We should take always take `activeMtx` before such an operation (this is already done).

Reply:
so we rely on the fact that lru cache lib does all the operations synchronously. well... ok...